src/Microsoft.Management.Infrastructure.CimCmdlets/Microsoft.Management.Infrastructure.CimCmdlets.csproj

```diff
@@ -6,6 +6,14 @@
     <AssemblyName>Microsoft.Management.Infrastructure.CimCmdlets</AssemblyName>
   </PropertyGroup>
 
+   <ItemGroup>
+    <EmbeddedResource Update="resources/*.resx">
+      <StronglyTypedClassName>Strings</StronglyTypedClassName>
+      <StronglyTypedNamespace>Microsoft.Management.Infrastructure.CimCmdlets</StronglyTypedNamespace>
+      <ManifestResourceName>$(RootNamespace).resources.%(EmbeddedResource.StronglyTypedClassName)</ManifestResourceName>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
why do we need extra changes to `CimCmdlet.csproj`?

Reply:
Resx file name is in another pattern :-(  My intention in the PR was to change only csproj files.
We could rename the file and use common csproj pattern.

Reply:
Got it. We talked (offline) about the names of the resource streams we used and agreed that they should be fixed if we go with this change, but should be a separate PR.

Reply:
We could rename CimCmdlet resx files before the PR so that we can remove the workaround from the PR.

Reply:
That sounds fine, but not sure if the ResGen code needs to be updated for fixing the name.

Reply:
Done in #12955.

Reply:
That PR was merged.

Reply:
The workaround is removed.